### PR TITLE
Add all Ingress-nginx controller packages

### PR DESCRIPTION
**Pull request by strongjz**
Add packages required for building ingress-nginx 

#### For new package PRs only

- [ ] This PR is marked as fixing a pre-existing package request bug
- [ ] Alternatively, the PR is marked as related to a pre-existing package request bug, such as a dependency
- [x] REQUIRED - The package is available under an OSI-approved or FSF-approved license
- [ ] REQUIRED - The version of the package is still receiving security updates
- [x] REQUIRED - The package is added to `packages.txt`

#### For security-related PRs

- [ ] The security fix is recorded in `advisories` and `secfixes`

### CODE REVIEW

File: msgpack.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
+package:
+  name: msgpack
+  version: 3.3.0
+  epoch: 0
+  description: "msgpack for C/C++."
+  copyright:
+    - license: Boost Software License - Version 1.0
```
Comment (strongjz) [inline msgpack.yaml:7]: @kaniini are we concerned about this license?

Comment (kaniini) [reply msgpack.yaml:7]: No, its a free license, its fine.  Use SPDX tag `BSL-1.0`.

File: jaeger-cpp.yaml [hunk 2]
```diff
@@ -0,0 +1,75 @@
...
+  name: jaeger-cpp
+  version: 0.7.0
+  epoch: 0
+  description: "Jaeger SDK with OpenTracing API for C++ binding."
+  target-architecture:
+    - all
+  copyright:
+    - paths:
+        - "*"
+      license: MIT license
```
Comment (kaniini) [inline jaeger-cpp.yaml:11]:
```suggestion
      license: MIT
```

File: opentracing.yaml [hunk 3]
```diff
@@ -0,0 +1,38 @@
+package:
+  name: opentracing
+  version: f86b33f3d9e7322b1298ba62d5ffa7a9519c4c41
+  epoch: 0
+  description: "C++ implementation of the OpenTracing API http://opentracing.io"
+  copyright:
+    - license: Apache-2.0 license
```
Comment (kaniini) [inline opentracing.yaml:7]:
```suggestion
    - license: Apache-2.0
```

File: opentracing.yaml [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    runtime:
+
+environment:
+  contents:
+    packages:
+      - wolfi-base
+      - busybox
+      - ca-certificates-bundle
+      - build-base
+      - ca-certificates-bundle
```
Comment (kaniini) [inline opentracing.yaml:18]:
```suggestion
```

File: zipkin-cpp.yaml [hunk 5]
```diff
@@ -0,0 +1,46 @@
+package:
+  name: zipkin-cpp
+  version: f69593138ff84ca2f6bc115992e18ca3d35f344a
+  epoch: 0
+  description: "OpenTracing implementation for Zipkin in C++."
+  copyright:
+    - license: Apache-2.0 license
```
Comment (kaniini) [inline zipkin-cpp.yaml:7]:
```suggestion
    - license: Apache-2.0
```

File: ingress-nginx-controller.yaml [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+      -o "${{targets.destdir}}/nginx-ingress-controller" ${PKG}/cmd/nginx
+
+      setcap cap_net_bind_service=+ep ${{targets.destdir}}/nginx-ingress-controller \
+      && setcap -v cap_net_bind_service=+ep ${{targets.destdir}}/nginx-ingress-controller
+
+update:
+  enabled: true
+  github:
+    identifier: kubernetes/ingress-nginx
+    ddstrip-prefix: controller-v
```
Comment (jdolitsky) [inline ingress-nginx-controller.yaml:73]: yo dawg

**Comment by strongjz** [comment]
https://github.com/wolfi-dev/os/pull/2554 is needed to update wolfictl to support vars with underscores.

**Review by imjasonh — DISMISSED** [review]

**Review by imjasonh — APPROVED** [review]

**Review by imjasonh — DISMISSED** [review]

File: datadog-cpp.yaml [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+      - opentracing-cpp
+      - msgpack
+
+pipeline:
+  - uses: fetch
+    with:
+      uri: https://github.com/DataDog/dd-opentracing-cpp/archive/v${{package.version}}.tar.gz
+      expected-sha256: 586f92166018cc27080d34e17c59d68219b85af745edf3cc9fe41403fc9b4ac6
+      strip-components: 1
+
```
Comment (imjasonh) [inline datadog-cpp.yaml:31]:
```suggestion
```

**Review by imjasonh — APPROVED** [review]

**Pull request merged**